launch/jdt.ls.remote.server.launch

```diff
@@ -42,10 +43,10 @@
     <setAttribute key="selected_target_bundles">
         <setEntry value="ch.qos.logback.classic@default:default"/>
         <setEntry value="ch.qos.logback.core@default:default"/>
-        <setEntry value="ch.qos.logback.slf4j@default:false"/>
-        <setEntry value="com.google.gson@default:default"/>
+        <setEntry value="com.google.gson*2.8.9.v20220111-1409@default:default"/>
```

Comment:
I see we're duplicating gson because lsp4j 0.12.0 requires gson `[2.8.0, 2.9.0)` while m2e.jdt requires gson `[2.9.1,3.0.0)` ? Just want to confirm. I think it should be safe since it's just a library, but annoying. Hopefully we can finally bump lsp4j in the new year.

Reply:
You are right.